change/@ni-nimble-blazor-06621d14-f2d1-455b-84b3-263be9c39942.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I tried building your branch on macOS and unfortunately `npm run build` now fails. I'm not sure why it fails on my mac but succeeds on the GitHub Actions Linux agents.

Can you look at disabling the BlazorHybrid project build on macOS?

```
Build FAILED.

/usr/local/share/dotnet/sdk/6.0.301/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): error NETSDK1100: Windows is required to build Windows desktop applications. [/Users/jesse/Documents/GitHub/nimble2/packages/nimble-blazor/Examples/Demo.Hybrid/Demo.Hybrid.csproj]
    0 Warning(s)
    1 Error(s)
```

Reply:
Or maybe [this SO answer will work for us](https://stackoverflow.com/a/73038414). I can try it out.

Reply:
@jattasNI just to double check, based on the SO comment you need a dotnet 6.0.4xx version, is the dotnet version on your mac compatible with that?

Reply:
Upgrading from 6.0.301 to 6.0.408 fixes the build failure with no additional code changes. Can you update the minimum .NET version in the nimble-blazor CONTRIBUTING? 

We should also ping other people with macs to let them know it's time to update. @fredvisser and @m-akinc are the ones I'm aware of with a mac as primary or secondary dev machine. Go [here](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) and run the latest installer for your chipset and ensure `dotnet --version` says `6.0.4xx`!

Reply:
As an aside:
> Starting with net6 SDK 6.0.4xx, it will be possible to build WPF/WinForms projects on non-windows operating systems.

that's pretty cool 😎

Reply:
Actually @jattasNI Microsoft says Blazor Hybrid apps should full-on work for macos: https://learn.microsoft.com/en-us/aspnet/core/blazor/supported-platforms?view=aspnetcore-7.0

Reply:
Updated CONTRIBUTING docs.